.github/workflows/tools-sync.yml

```
@@ -0,0 +1,32 @@
+name: tools-sync
+
+on: 
+  schedule:
+    - cron: "0 0 * * *"
```

I would also add a manual trigger:

```suggestion
on: 
  workflow_dispatch:
  schedule:
    - cron: "0 0 * * *"
```

so we can also trigger it manually if we want

---

good idea